pysces/core2/PyscesCore2.py

```diff
@@ -374,6 +374,7 @@ def setArg(self, var, value=None):
         self.args.append(var)
 
     def addFormula(self, formula):
+        print('PC2 377:', formula)
```

Comment:
This looks like a debug print statement that should either be commented or put into an if-loop for debug mode.

Reply:
thought I had all these, anyway comment out for now